dev/ory/kratos.yml

```diff
@@ -77,11 +77,23 @@ selfservice:
       after:
         password:
           hooks:
+            - hook: web_hook
+              config:
+                url: http://e2e-tests:4012/kratos/registration
+                method: POST
+                can_interrupt: true
+                body: file:///home/ory/body.jsonnet # TODO: use a base64 encoding instead
```

Comment:
this should include what is in https://github.com/GaloyMoney/galoy/blob/main/dev/ory/body.jsonnet